### PR TITLE
Dependencies.

### PR DESCRIPTION
**Pull request by Dawn-used-yeet**
a

### CODE REVIEW

File: .github/workflows/beta.yml [hunk 1]
```diff
@@ -96,7 +95,7 @@ jobs:
       if: ${{ env.SKIP_BUILD != 'true' }}
       run: chmod +x ./gradlew
 
-    - name: Build with Gradle
+    - name: Build with Grad le
```
Comment (rebelonion) [inline .github/workflows/beta.yml:98]: typo

File: .github/workflows/beta.yml [hunk 2]
```diff
@@ -38,30 +39,28 @@ jobs:
     - name: Get Commits Since Last Run
       run: |
         if [ -f last_sha.txt ]; then
-          LAST_SHA=$(cat last_sha.txt)
+        LAST_SHA=$(cat last_sha.txt)
```
Comment (rebelonion) [inline .github/workflows/beta.yml:42]: ?

File: .github/workflows/beta.yml (hunk 1 above)
Comment (Dawn-used-yeet) [reply .github/workflows/beta.yml:98]: Change it.

File: .github/workflows/beta.yml (hunk 2 above)
Comment (Dawn-used-yeet) [reply .github/workflows/beta.yml:42]: Should I revert it?

Comment (Dawn-used-yeet) [reply .github/workflows/beta.yml]: I changed it. Does the same thing.

**Comment by Dawn-used-yeet** [comment]
Rintaro fixed already

**Pull request closed without merging**